### ISSUE TITLE
Xcode Beta 3 protocol conformance error

### ISSUE THREAD

**Issue by nathanbroyles** [appcues/appcues-ios-sdk#549]
Using Xcode 16 Beta 3 and Appcues 3.2 I get a `URLOpening` protocol conformance error in `UIApplication+TopViewController.swift`.

<img width="787" alt="Screenshot 2024-07-09 at 9 26 49 AM" src="https://github.com/appcues/appcues-ios-sdk/assets/1165514/8fa4c3e7-e591-4967-901e-c7d79d53bcb8">

**Comment by mmaatttt** [comment]
Hi @nathanbroyles, thanks for the heads up. We've been reviewing compatibility with Xcode 16 and iOS 18, and we'll be sure to have a fully compatible release before the official releases of Xcode 16 and iOS 18.

**Comment by dcramps** [comment]
> Hi @nathanbroyles, thanks for the heads up. We've been reviewing compatibility with Xcode 16 and iOS 18, and we'll be sure to have a fully compatible release before the official releases of Xcode 16 and iOS 18.

do you have a branch for this? we're blocked on our own xcode 16/ios 18 updates until we either fork appcues and fix this ourselves or point at an appcues development branch that has the fix

**Comment by mmaatttt** [comment]
Hi @dcramps, @nathanbroyles,

We've just released 3.3.0 which fixes this compile error. We're still in the process of evaluating support for iOS 18 (no known issues so far, but we want to be thorough—please create new a new issue if you encounter any bugs), so we don't consider this version official support for iOS 18, but it will allow you to proceed with your own update process.

Cheers,
Matt
